addons/project/models/project.py

```diff
@@ -1392,6 +1402,21 @@ def action_subtask(self):
     # Actions
     # ------------
 
+    def action_open_task_form(self):
+        """ Open the form view of the task it depends on
+
+            The self should contains the task which the current task depends on it in the view tree of depend_on_ids field.
+            That is the task in the line in which the user clicks on the button 'View Task'.
+        """
```

Comment:
Should we keep this or just "Opens the form view of the task" ?

Reply:
We can just put this comment. In fact, I put it to explain the using of this method. If you prefer, I can either put your comment or remove the comment in this method.